CMakeLists.txt

```diff
@@ -144,6 +148,7 @@ option(ENABLE_HEAVY_LOGGING "Should heavy debug logging be enabled" ${ENABLE_HEA
 option(ENABLE_HAICRYPT_LOGGING "Should logging in haicrypt be enabled" 0)
 option(ENABLE_SHARED "Should libsrt be built as a shared library" ON)
 option(ENABLE_STATIC "Should libsrt be built as a static library" ON)
+option(ENABLE_PKTINFO "Enable target IP address extraction on listner" ${ENABLE_PKTINFO_DEFAULT})
```

Comment:
```suggestion
option(ENABLE_PKTINFO "Enable using IP_PKTINFO to allow the listener extracting the target IP address from incoming packets" ${ENABLE_PKTINFO_DEFAULT})
```

Reply:
Maybe just use `OFF` here?

Reply:
We may decide in the future to turn it to ON on all supporting platforms, and this way it will be only OFF on platforms that do not support it.

Note also that this IS supported on Windows, just the procedure of sending and receiving UDP packets is using a function that doesn't allow the use of CMSG. To enable that it would have to be reimplemented.